import java.util.Scanner;

class TwoDToOneD {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Enter number of rows: ");
        int rows = input.nextInt();

        System.out.print("Enter number of columns: ");
        int columns = input.nextInt();

        int[][] matrix = new int[rows][columns];

        System.out.println("Enter matrix elements:");
        for (int i = 0; i < matrix.length; i++)
            for (int j = 0; j < matrix[i].length; j++)
                matrix[i][j] = input.nextInt();

        int[] array = new int[rows * columns];
        int index = 0;

        for (int i = 0; i < matrix.length; i++)
            for (int j = 0; j < matrix[i].length; j++)
                array[index++] = matrix[i][j];

        System.out.print("1D Array: ");
        for (int i = 0; i < array.length; i++)
            System.out.print(array[i] + " ");

        input.close();
    }
}
